cmd/traffic_layout/traffic_layout.cc

```diff
@@ -252,7 +253,7 @@ main(int argc, const char **argv)
   // check for traffic_runroot operations
   for (int i = 0; i < argc; i++) {
     if (!strcmp(argv[i], "--init") || !strcmp(argv[i], "--remove") || !strcmp(argv[i], "-i") || !strcmp(argv[i], "-r") ||
-        !strcmp(argv[i], "--force")) {
+        !strcmp(argv[i], "--force") || !strcmp(argv[i], "--relocate")) {
```

Comment:
This could be better with string literals...
 ie Think you should be able to say:
"--force"_sv != argv[i] || "--relocate"_sv != argv[i]